### PR TITLE
Remove near-meta role openshift_cli_facts

### PR DESCRIPTION
**Pull request by michaelgugino**
openshift_cli_facts only runs a single task, and otherwise
the only existence is to serve as a link in the dependency
chain towards openshift_facts.

This commit flattens openshift_cli into one role.

### CODE REVIEW

**Comment by michaelgugino** [comment]
bot, retest this please

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
It looks like openshift_version depends on openshift.common.cli_image being set but I don't see that role actually depending on this.

File: roles/openshift_cli/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---
+- openshift_facts:
+    role: common
+    local_facts:
+      cli_image: "{{ osm_image | default(None) }}"
```
Comment (kwoodson) [inline roles/openshift_cli/tasks/main.yml]: Is it possible to move this to `playbooks/common/openshift-cluster/initialize_facts.py`?  It seems to me that osm_image is coming from the inventory and we could just assign this to the rest of the facts we are setting in a single place.  

@michaelgugino, thoughts on this?

**Comment by kwoodson** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
flake https://github.com/openshift/origin/issues/15977

**Pull request merged**